### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by smoitozo** [Eduix/crowd-shibboleth-module#2]
I've been working with the shibboleth-filter and shibboleth-filter-config modules this week and have written some documentation for newbies (like me) who need to know how to build it. Can I contribute the documentation to the project?

Also, I have some questions:
- Is the nordunet-sso module  required in addition to the shibboleth-filter? 
- I have crowd SAMLized with the shibboleth-filter but JIRA doesn't seem to want to engage in SSO with Crowd. Is that because I don't have the nordunet-sso plugin installed? How do I install it, the documentation says to place it in a location that does not exist in the latest version of Crowd.

**Comment by sebValiantys** [comment]
Hi all,

Seeing people have been successful in configuring JIRA's authentication through Crowd and Shibboleth gives me hope.

I have correctly managed to configure crowd's authentication. Login goes through our dedicated authentication page and access to crowd works smoothly. When I try to login to JIRA though, I keep getting redirected to Crowd.

I went through the settings several times already and I keep missing the cause. 
@smoitozo if by any chance, you remember what problems you faced and what corrections you made ? 

When I look at the network in my browser I see the following stack :
 1 : my-jira-url
 2 : my-crowd-url/plugins/servlet/ssocookie?redirectTo={jira's dashboard}
 3 : my-crowd-url/console/login.action?error=true
 4 : my-crowd-url/console/defaultstartpage.action
 5 : my-crowd-url/console/secure/console.action

I did find errors in the logs. A LazyInitializationException happening within crowd and resulting in a failed authentication. First it was on the call to CrowdUserDetailsService.loadUserByName (in ShibbolethSSOFilter.attemptAuthentication) which I changed with a call to UserManager.getUser since we don't care about the extra data retrieved by the first method. But unfortunately, later on another call is made to CrowdUserDetailsService.loadUserByToken through the call to getAuthenticationManager().authenticate(crowdAuthRequest). At this point I don't have any means to avoid the exception.

I haven't been able to figure out why I would get this exception when login into JIRA a not Crowd even though it is the exact same code running both times. 

Any hints will be greatly appreciated.
